icu4j/main/classes/core/src/com/ibm/icu/number/Precision.java

```diff
@@ -20,6 +20,7 @@
  *
  * @stable ICU 62
  * @see NumberFormatter
+ * @internal
```

Comment:
Is this really @internal? It seems to me to be intended for public use.

The "public" flags for constructors and "createCopy" methods below are a bit inconsistent, but I'm restoring them to what they were. I don't think we need to touch this file at all?

Reply:
Yeah, this should not be `@internal`.  It is already `@stable`.